packages/flutter/lib/src/foundation/stack_frame.dart

```diff
@@ -82,28 +82,38 @@ class StackFrame {
         .toList();
   }
 
-  static StackFrame? _parseWebFrame(String line) {
+  /// Parses a single [StackFrame] from a line of a [StackTrace].
+  ///
+  /// Returns null if format is not as expected.
+  static StackFrame? _treyParseWebFrame(String line) {
```

Comment:
If we are going to rename it,

```suggestion
  static StackFrame? _tryParseWebFrame(String line) {
```

I think we should either keep the existing name pattern, or also rename `_parseWebNonDebugFrame` for consistency within this file.